### PR TITLE
Set KeepAliveTimeout to 620000 (620secs)

### PR DESCRIPTION
**Pull request by pengin7384**
- Fix 502 error

### CODE REVIEW

File: client/index.js [hunk 1]
```diff
@@ -280,11 +280,13 @@ app.get('/get_address', (req, res, next) => {
 
 // We will want changes in ports and the database to be broadcast across
 // all instances so lets pass this info into the p2p server
-app.listen(PORT, () => {
+const server = app.listen(PORT, () => {
   logger.info(`[${CLIENT_PREFIX}] App listening on port ${PORT}`);
   logger.info(`[${CLIENT_PREFIX}] Press Ctrl+C to quit.`);
 });
 
+server.keepAliveTimeout = 620000;
```
Comment (liayoo) [inline client/index.js:288]: Could you also set headersTimeout to be greater than keepAliveTimeout? (related issue: https://github.com/nodejs/node/issues/27363)

Comment (pengin7384) [reply client/index.js:288]: Thanks! Done.

**Comment by pengin7384** [comment]
PTAL! @liayoo

**Comment by liayoo** [comment]
LGTM. Thanks for fixing the issue!

**Comment by pengin7384** [comment]
PTAL! @platfowner

File: client/index.js [hunk 2]
```diff
@@ -280,11 +280,14 @@ app.get('/get_address', (req, res, next) => {
...
```
Comment (platfowner) [inline client/index.js:288]: Does this value have any special meaning? It seems that some articles (like https://shuheikagawa.com/blog/2019/04/25/keep-alive-timeout/) suggest some values 61 * 1000.

And what about adding some comments about the unit? e.g.
server.keepAliveTimeout = 62 * 1000;  // 62 seconds

Comment (pengin7384) [reply client/index.js]: Google load balancer has a tcp timeout of 600 seconds.([link](https://cloud.google.com/load-balancing/docs/https#timeouts_and_retries)) keepAliveTimeout should be set to a value greater than this. In fact, for the tracker server, load balancing has not been applied yet, so it is not necessary to set it up yet.

Comments are added!

**Pull request merged**